### ISSUE TITLE
Feedback

### ISSUE THREAD

**Issue by troyshu** [cuny-sps-msda-data622-2017fall/data622hw2#1]
@sunwsusan Sorry I'm late! Emails about Slack notifications went to my Updates tab in Gmail, so didn't see them.

My thoughts:

 - under score_model.py: what is meant by "model accuracy report"? is it sklearn's [classification report](http://scikit-learn.org/stable/modules/generated/sklearn.metrics.classification_report.html)? or are they supposed to come up with their own "model performance report" using different eval metrics and plots (and justify why they chose those eval metrics/plots, which would help them test their own understanding)?

 - I like the idea of having them try to convince us why they did certain data pre-processing steps; similarly to my point before, I think it helps them test their own understanding. It looks like this is part of the optional "eda.ipynb" step, but it might be educational to them to at least make the "justification" aspect required, whether that's through the comments in train_model.py, another question in the "Critical Thinking" section, or something else.

**Comment by sunwsusan** [comment]
Done and done.  Thank you!
